examples/human_example/human_example.gltf

```diff
@@ -0,0 +1,2335 @@
+			"byteOffset":1233068,
+			"target":34963
+		},
+		{
+			"buffer":0,
+			"byteLength":4800,
+			"byteOffset":1246412,
+			"target":34962
+		},
+		{
+			"buffer":0,
+			"byteLength":4800,
+			"byteOffset":1251212,
+			"target":34962
+		},
+		{
+			"buffer":0,
+			"byteLength":3200,
+			"byteOffset":1256012,
+			"target":34962
+		},
+		{
+			"buffer":0,
+			"byteLength":1600,
+			"byteOffset":1259212,
+			"target":34962
+		},
+		{
+			"buffer":0,
+			"byteLength":6400,
+			"byteOffset":1260812,
+			"target":34962
+		},
+		{
+			"buffer":0,
+			"byteLength":4320,
+			"byteOffset":1267212,
+			"target":34963
+		},
+		{
+			"buffer":0,
+			"byteLength":6960,
+			"byteOffset":1271532,
+			"target":34962
+		},
+		{
+			"buffer":0,
+			"byteLength":6960,
+			"byteOffset":1278492,
+			"target":34962
+		},
+		{
+			"buffer":0,
+			"byteLength":4640,
+			"byteOffset":1285452,
+			"target":34962
+		},
+		{
+			"buffer":0,
+			"byteLength":2320,
+			"byteOffset":1290092,
+			"target":34962
+		},
+		{
+			"buffer":0,
+			"byteLength":9280,
+			"byteOffset":1292412,
+			"target":34962
+		},
+		{
+			"buffer":0,
+			"byteLength":5424,
+			"byteOffset":1301692,
+			"target":34963
+		},
+		{
+			"buffer":0,
+			"byteLength":4896,
+			"byteOffset":1307116,
+			"target":34962
+		},
+		{
+			"buffer":0,
+			"byteLength":4896,
+			"byteOffset":1312012,
+			"target":34962
+		},
+		{
+			"buffer":0,
+			"byteLength":3264,
+			"byteOffset":1316908,
+			"target":34962
+		},
+		{
+			"buffer":0,
+			"byteLength":1632,
+			"byteOffset":1320172,
+			"target":34962
+		},
+		{
+			"buffer":0,
+			"byteLength":6528,
+			"byteOffset":1321804,
+			"target":34962
+		},
+		{
+			"buffer":0,
+			"byteLength":4080,
+			"byteOffset":1328332,
+			"target":34963
+		},
+		{
+			"buffer":0,
+			"byteLength":30204,
+			"byteOffset":1332412,
+			"target":34962
+		},
+		{
+			"buffer":0,
+			"byteLength":30204,
+			"byteOffset":1362616,
+			"target":34962
+		},
+		{
+			"buffer":0,
+			"byteLength":20136,
+			"byteOffset":1392820,
+			"target":34962
+		},
+		{
+			"buffer":0,
+			"byteLength":10068,
+			"byteOffset":1412956,
+			"target":34962
+		},
+		{
+			"buffer":0,
+			"byteLength":40272,
+			"byteOffset":1423024,
+			"target":34962
+		},
+		{
+			"buffer":0,
+			"byteLength":28128,
+			"byteOffset":1463296,
+			"target":34963
+		},
+		{
+			"buffer":0,
+			"byteLength":25812,
+			"byteOffset":1491424,
+			"target":34962
+		},
+		{
+			"buffer":0,
+			"byteLength":25812,
+			"byteOffset":1517236,
+			"target":34962
+		},
+		{
+			"buffer":0,
+			"byteLength":17208,
+			"byteOffset":1543048,
+			"target":34962
+		},
+		{
+			"buffer":0,
+			"byteLength":8604,
+			"byteOffset":1560256,
+			"target":34962
+		},
+		{
+			"buffer":0,
+			"byteLength":34416,
+			"byteOffset":1568860,
+			"target":34962
+		},
+		{
+			"buffer":0,
+			"byteLength":25788,
+			"byteOffset":1603276,
+			"target":34963
+		}
+	],
+	"buffers":[
+		{
+			"byteLength":1629064,
+			"uri":"Human_Example.bin"
```

Comment:
The file name is wrong here. This would be a simple fix, if you are sure you uploaded the correct files. Just want to make sure.